build/tutorials-contest/liquid-avax/Backend/backend.js

```diff
@@ -0,0 +1,272 @@
+// to the user-chosen node.
+
+import fs from "fs";
+import {Avalanche, BinTools, BN} from "avalanche";
+import Web3 from "web3";
+import { Buffer } from "buffer";
+
+let xChain, xKeyChain, xChainAddress;
+let cChain, cKeyChain, cChainAddress;
+let pChain, pKeyChain, pChainAddress;
+
+let xChainBlockchainID, cChainBlockchainID, pChainBlockchainID;
+
+let web3;
+
+let contractAbi;
+
+let masterAddress = "0x7bD7A7D2Ba70db40740780828b236F9246BB7F78";
+let privateKey = JSON.parse(await fs.readFileSync("./data.json")).privateKey;
+
+let currentTxId = [];
+
+let utxoset;
+
+
+async function waitForStaked() {
+    let stakingContract = new web3.eth.Contract(contractAbi, "0x5bD27eF83d57915FC3eE7feB2FebEB9c69d52B04");
+    let stakedEvent = stakingContract.events.Staked();
+    await web3.eth.subscribe('logs', {
+        address: stakedEvent.arguments[0].address,
+        topics: stakedEvent.arguments[0].topics,
+    }, async function (error, logs) {
+        if (!error) {
+            console.log("New stake")
+            let dataHex = logs.data.substring(2);
+            let id = parseInt(dataHex.substring(0, 64), 16);
+            let amountWithDecimals = parseInt(dataHex.substring(64), 16);
+            //Withdraw amount staked
+            web3.eth.defaultAccount = masterAddress
+            let data = stakingContract.methods.withdraw(id).encodeABI();
+            let nonce = await web3.eth.getTransactionCount(masterAddress, "pending");
+            let tx = {from:masterAddress, to:"0x5bD27eF83d57915FC3eE7feB2FebEB9c69d52B04", data:data, gasPrice:225*10**9, gas:2100000, nonce:nonce}
+            let stx = await web3.eth.accounts.signTransaction(tx, privateKey);
+            let htx = await web3.eth.sendSignedTransaction(stx.rawTransaction);
+            console.log(htx)
+            await CtoP(id, amountWithDecimals);
+        } else {
+            console.log(error)
+        }
+    })
+}
+
+async function stakeToNode(nodeId) {
+    
+}
+
+async function CtoP() { //a C --> P cross-chain transfer doesn't exists, but C --> X, X --> P does.
+    
+    // Includes the fees in the transferred AVAX
+    
+    xChainAddress = xKeyChain.getAddressStrings();
+    cChainAddress = cKeyChain.getAddressStrings();
+    pChainAddress = pKeyChain.getAddressStrings();
+    
+    let amount = new BN("1000000000"); //1 AVAX
+    
+    let cChainHexAddress = "0x7bD7A7D2Ba70db40740780828b236F9246BB7F78";
+    
+    let nonce = await web3.eth.getTransactionCount(cChainHexAddress, "pending");
+    
+    let unsignedCtoXTx = await cChain.buildExportTx(
+        amount,
+        cAvaxAssetId,
+        xChainBlockchainID,
+        cChainHexAddress,
+        cChainAddress[0],
+        xChainAddress,
+        nonce,
+    )
+    
+    let signedCtoXTx = await unsignedCtoXTx.sign(cKeyChain);
+    
+    let CtoXTxId = await cChain.issueTx(signedCtoXTx);
+    
+    console.log("CtoXTxId " + CtoXTxId);
```

Comment:
Use interpolation instead of concatenation

```ts
console.log(`CtoXTxId ${CtoXTxId}`)
```

Also remove all `console.log` statements

Reply:
What should replace it with ? Just remove all logging, except for error with which I should use `throw` ?